app/models/claim/database_of_qualified_teachers_report_request.rb

```diff
@@ -0,0 +1,24 @@
+require "csv"
+
+class Claim
+  class DatabaseOfQualifiedTeachersReportRequest
+    ATTRIBUTES = {
+      reference: "Claim reference",
+      teacher_reference_number: "Teacher reference number",
+    }.freeze
+
+    def initialize(claims)
+      @claims = claims
+    end
+
+    def to_csv
+      CSV.generate(headers: true) do |csv|
```

Comment:
Does the `:headers` option actually do anything on the `generate` method?

```suggestion
      CSV.generate do |csv|
```

Reply:
I thought it did, but it clearly doesn't here.